### ISSUE TITLE
Typo in linchpin validate 

### ISSUE THREAD

**Issue by samvarankashyap** [CentOS-PaaS-SIG/linchpin#741]
When I run 
```
linchpin validate
```
output:
```
(lp_dep) 🎩linchpin validate
[ERROR] topology valid
layout valid
```
Though the topology is valid its spits out ERROR

**Comment by herlo** [comment]
#733 should have fixed this issue. Please rebase against develop (as it's been merged already) and reopen if it persists.

**Comment by samvarankashyap** [comment]
Re-based error still exists !

**Comment by 14rcole** [comment]
@samvarankashyap I can't reproduce the error anymore:

```bash
(linchpin) [ryan@dhcp40-146 aws]$ linchpin validate
[SUCCESS] Topology for target 'aws-ec2-new' is valid
[SUCCESS] Topology for target 'aws-sg-new' is valid
[SUCCESS] Topology for target 'aws-s3-new' is valid
[SUCCESS] Topology for target 'aws-ec2-key-new' is valid

```

**Comment by samvarankashyap** [comment]
@ryan could you try it on dummy topology ?

**Comment by samvarankashyap** [comment]
@14rcole : Fresh installation of linchpin on develop branch : 
```
(dummywk) 🎩mkdir workspace
(dummywk) 🎩ls
linchpin  workspace
(dummywk) 🎩cd workspace/
(dummywk) 🎩ls
(dummywk) 🎩linchpin init
PinFile and file structure created at /home/srallaba/workspace/lp_ws/dummywk/workspace
(dummywk) 🎩ls
credentials  hooks  inventories  layouts  linchpin.conf  linchpin.log  localhost  PinFile  topologies
(dummywk) 🎩linchpin validate
[ERROR] topology valid
layout valid
```

**Comment by samvarankashyap** [comment]
fixed by #750
